var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -101,7 +101,7 @@ def patch(self):
 
     @property
     def common_config_args(self):
-        return [
+        config_args = [
             '-prefix', self.prefix,
```

Comment:
Here is the first run output using:

    @when('@3')
    def configure(self):
        configure('-fast',
                  '-thread',
                  *self.common_config_args)

PARTIAL OUPTUT:

See build log for details:
  /tmp/jeg/spack-stage/spack-stage-9YDAba/qt-x11-free-3.3.8b/spack-build.out

/home/jeg/work_spack/spack/var/spack/repos/builtin/packages/qt/package.py:126, in configure:
     122      @when('@3')
     123      def configure(self):
     124          configure('-fast',
     125                    '-thread',
  >> 126                    *self.common_config_args)
==> Error: Installation process had nonzero exit code.
[jeg@localhost packages]$ cat /tmp/jeg/spack-stage/spack-stage-9YDAba/qt-x11-free-3.3.8b/spack-build.out
==> ./configure -fast -thread -prefix /home/jeg/work_spack/spack/opt/spack/Linux-x86_64/gcc-4.8.3/qt-3.3.8b-alld534dmzulshzoqgzgrulpoy7ux7d2 -v -opensource -opengl -release -shared -confirm-license -openssl-linked -dbus-linked -optimized-qmake -no-openvg -no-pch -no-nis
-opensource: unknown argument
-opengl: unknown argument
-confirm-license: unknown argument
-openssl-linked: unknown argument
-dbus-linked: unknown argument
-optimized-qmake: unknown argument
-no-openvg: invalid command-line switch
IBASE auto-detection...
  Could not find libgds.* anywhere in /lib /usr/lib
IBASE disabled.
MySQL auto-detection...
  Could not find libmysqlclient.* anywhere in /lib /usr/lib
MySQL disabled.
ODBC auto-detection...
  Could not find libodbc.* anywhere in /lib /usr/lib
ODBC disabled.
PostgreSQL auto-detection...
  Could not find libpq.* anywhere in /lib /usr/lib
PostgreSQL disabled.
CUPS auto-detection... ()
  Found cups lib in /usr/lib
  Found cups/cups.h in /usr/include
CUPS enabled.
OpenGL auto-detection... ()
  Found GLU lib in /usr/lib